### DISCUSSION TITLE
Tweepy through academic account

### DISCUSSION THREAD

**Discussion by tlerksuthirat** [tweepy/tweepy#2040] [Questions] [answered]
I am using the tweepy to get the tweets through the academic account, I am wondering whether the academic account can retrieve all the public tweets or it is actually get the portions?

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what exactly you're asking, but full-archive search is available with Academic Research access.

**Reply by tlerksuthirat** [reply]
Thank you.
